src/main/TeamDAO.java

```diff
@@ -0,0 +1,29 @@
+package main;
+
+import lombok.SneakyThrows;
+import org.springframework.stereotype.Repository;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.util.LinkedList;
+import static main.DBDateStore.connection;
+
+@Repository
+public class TeamDAO {
+    private static final String SELECT_TEAMS = "SELECT id, team_name, horse, rider FROM teams;";
+
+    @SneakyThrows
+    public LinkedList<Team> getTeams() {
+        LinkedList<Team> teams = new LinkedList<>();
```

Comment:
1) Always use geric interface (`List` in this case), user of your method should not care which type of implementation of collection is used.
But when you care, you can just cast to needed type in place of usage.